contributing.md

```diff
@@ -1,14 +1,51 @@
-#### How to reach us
-The mailing list for developers : systemroles@lists.fedorahosted.org
 
-[Subscribe to the mailing list here](https://lists.fedorahosted.org/admin/lists/systemroles.lists.fedorahosted.org/)
+# Contributing to Network @ Linux System Roles
 
-[Archive of the mailing list](https://lists.fedorahosted.org/archives/list/systemroles@lists.fedorahosted.org/)
+## Where to start
+- **Bugs and needed implementations** are listed on [Github
+  Issues](https://github.com/linux-system-roles/network/issus). Issues labeled
+with **help wanted** are likely to be suitable for new contributors!
+- **Code** is managed on
+  [Github](https://github.com/linux-system-roles/network), using [Pull
+Requests](https://help.github.com/en/github/collaborating-with-issues-and-pull-requests/about-pull-requests).
 
-If you are using IRC, join the `#systemroles` IRC channel on freenode
+## How to contribute
+
+1. Make a
+   [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo)
+of this repository.
+
+2. Create a new git branch on your local fork (the name is not relevant) and
+   make the changes you need to complete an issue.
+
+3. Do not forget to run unit and integration tests before pushing any changes!
+  
+  - This project uses [tox](https://tox.readthedocs.io/en/latest/) to run unit
+    tests. You can try it with `tox -e py36` in case you want to try it using
+    Python 3.6.
+
+  - Format the code using [Python black](https://black.readthedocs.io/en/stable/)
+
+  - Check the YML files are correctly formatted using `tox -e yamllint`.
+
+  - Integration tests are executed as
+   [ansible-playbooks](https://docs.ansible.com/ansible/latest/user_guide/playbooks.html).
```

Comment:
It would be great to add an example using the standard test roles qemu inventory script.

Reply:
I added the link to the VM too. This is a really nice idea, thanks!